buildenv/jenkins/common/build.groovy

```diff
@@ -326,8 +326,45 @@ def archive_sdk() {
     }
 }
 
+def fetchFile(src, dest) {
+    def fileToMove = src.substring(src.indexOf('.') + 1)
+    sh "mv '//${fileToMove}' ${src}"
+    sh "cat ${src} >> ${dest}"
```

Comment:
Do we know why it works this way?

Reply:
There is a template for creating datasets in z/os and the name of the template appears after the string 'IEATDUMP success for DSN=' in the log content. There are multiple datasets created by the template in sequence. They have nearly the same name except the last part. For example, Jenkins.xxx.001, Jenkins.xxx.002, Jenkins.xxx.003. We concatenate those file in sequence and create a core.xxx.dmp for resultstore for triage.

Reply:
There is a template for creating datasets in z/os and the name of the template appears after the string 'IEATDUMP success for DSN=' in the log content. There are multiple datasets created by the template in sequence. They have nearly the same name except the last part. For example, Jenkins.xxx.001, Jenkins.xxx.002, Jenkins.xxx.003. We concatenate those file in sequence and create a core.xxx.dmp for resultstore for triage. Please let me know if there is any other question. Thanks!

Reply:
Please add this information as a comment in the code.

Reply:
> Please add this information as a comment in the code.

Good Morning! The comments were added. Thanks for checking it! @pshipton

Reply:
Has this been tested when there are actually two or more parts to the a core dump? If the reason for splitting the dataset into multiple parts was to avoid overly large files, this will bring that problem back.

Reply:
Hey @keithc-ca! Thanks for reviewing it! We only tested when there are only one part for the core dump.